clang/runtime/dpct-rt/include/device.hpp.inc

```diff
@@ -630,6 +668,24 @@ public:
     prop.set_global_mem_size(
         get_info<sycl::info::device::global_mem_size>());
     prop.set_local_mem_size(get_info<sycl::info::device::local_mem_size>());
+    if (this->has(sycl::aspect::ext_intel_memory_clock_rate))
```

Comment:
I think you need format this change, the code style is not aligned to the existing code.
"{" should not in a new line.